📝(content) Post: ADB over Wi-Fi and Ethernet

jkcinouye opened the pull request:
Content added to posts:
ADB over Wi-Fi and Ethernet

posts/adb-over-wifi.md

jkcinouye committed:
Add content to posts:

ADB over Wi-Fi and Ethernet

Snugug reviewed (changes requested):
Two items:
* This is Android documentation, should live in `android` instead of `posts`
* Images need to have their URLs updated

jkcinouye committed:
Added images to:

posts/adb-over-wifi.md

jkcinouye committed:
Renamed:

site/en/posts/adb-over-wifi.md ->
site/en/android-adb-over-wifi.md

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+1.  ADB over Wi-Fi
+1.  ADB over Ethernet
+
+For most phones and tablets, you can connect an Android device to your development machine using a USB-C cable and ADB will "just work." As of June 2023, most Chromebooks do not support ADB over USB, leaving the two other methods: Wi-Fi and Ethernet.
```
```suggestion
For most phones and tablets, you can connect an Android device to your development machine using a USB-C cable and ADB will "just work." As of August 2023, most Chromebooks do not support ADB over USB, leaving the two other methods: Wi-Fi and Ethernet.
```

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+**Note:** If your device does not have an Ethernet port, you can instead use an Ethernet dongle that connects to your device via USB and provides Ethernet capability.
+!!!
+
+![A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
+_A USB-A Ethernet dongle._
```
```suggestion
#[A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
```

We actually have a dedicated figure element specifically for this kind of setup. Replacing the `!` with a `#` should do it.

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+![A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
+_A USB-A Ethernet dongle._
+
+![A complete ADB over Ethernet setup including USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter.](ix://android/adb/double-ethernet.jpg)
+_A complete ADB over Ethernet setup, including: USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter._
```
```suggestion
#[A complete ADB over Ethernet setup including USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter.](ix://android/adb/double-ethernet.jpg)
```

Same here.

jkcinouye reviewed (commented):
Images added!
URL changed!

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+
+For most phones and tablets, you can connect an Android device to your development machine using a USB-C cable and ADB will "just work." As of June 2023, most Chromebooks do not support ADB over USB, leaving the two other methods: Wi-Fi and Ethernet.
+
+Note: You can also use [ADB over Bluetooth](https://developer.android.com/training/wearables/get-started/debugging) for WearOS devices
```
```suggestion
!!! aside.message--note
**Note:** You can also use [ADB over Bluetooth](https://developer.android.com/training/wearables/get-started/debugging) for WearOS devices
!!!
```

Making this a note popout

fuentespatrick commented:
The other ADB content lives here: https://chromeos.dev/en/android-environment/deploying-apps 
Do we want to keep them together and/or reference them from each other?

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+
+Note: You can also use [ADB over Bluetooth](https://developer.android.com/training/wearables/get-started/debugging) for WearOS devices
+
+# ADB over Wi-Fi
```
```suggestion
## ADB over Wi-Fi
```

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+**Tip:** For more information, see [Connect to a device over Wi-Fi](https://developer.android.com/tools/adb#wireless).
+!!!
+
+# ADB over Ethernet
```
```suggestion
## ADB over Ethernet
```

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+
+In some cases, it may not be possible to use a Wi-Fi network due to firewall restrictions, or a wired solution may simply be preferable. In these cases, you can connect over an Ethernet cable. The steps are similar to ADB over Wi-Fi but using an Ethernet cable.
+
+## Prerequisites
```
```suggestion
### Prerequisites
```

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+![A complete ADB over Ethernet setup including USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter.](ix://android/adb/double-ethernet.jpg)
+_A complete ADB over Ethernet setup, including: USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter._
+
+## Connecting ADB over Ethernet
```
```suggestion
### Connecting ADB over Ethernet
```

Snugug commented on site/en/android/adb-over-wifi.md:
```diff
@@ -0,0 +1,83 @@
+**Note:** These instructions assume your local network is running on the `192.168.1.x` address space.
+!!!
+
+!!! aside.message--alert
```
```suggestion
!!! aside.message--note
```

We don't have an "alert" type so this is rendering as a standard message, which isn't the vibe we're going for.

Snugug reviewed (changes requested):
Sorry, found a few more things when looking at the updated items actually rendered. These should all be click to fix.

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Update site/en/android/adb-over-wifi.md

Co-authored-by: Sam Richard <snugug@google.com>

jkcinouye committed:
Changes to be committed:
	renamed:    site/en/android/adb-over-wifi.md -> site/en/android-environment/adb-over-wifi.md

jkcinouye committed:
Changes to be committed:
	modified:   site/en/android-environment/adb-over-wifi.md

Weighting content in order to group ADB content.

jkcinouye committed:
Changes to be committed:
	renamed:    site/en/android-environment/adb-over-wifi.md -> site/en/android-environment/adb-over-wifi-and-ethernet.md
